src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
+  const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);
```

Comment:
What if post id was not provided? we really want to make an http request to another service with an invalid value? Before the request we should validate this value, and if it is not valid we should handle this (probably throw an error).